backup/syncer/job.go

```diff
@@ -160,7 +160,7 @@ func (s *jobSyncer) ensurePodSpec(in corev1.PodSpec) corev1.PodSpec {
 			s.backup.Namespace, s.backup.GetNameForJob(), backupToDir, DateTime)
 		// Add the check DiskUsage
 		// use expr because shell cannot compare float number
-		checkUsage := `[ $(expr $(df /backup|awk 'NR>1 {print $4}') \> $(du  /backup |awk 'END {if (NR > 1) {print $1 /(NR-1)} else print 0}')) -eq '1' ] || { echo disk available may be too small; exit 1;};`
+		checkUsage := `[ $(echo "$(df /backup|awk 'NR>1 {print $4}') > $(du  /backup |awk 'END {if (NR > 1) {print $1 /(NR-1)} else print 0}')"|bc) -eq '1' ] || { echo disk available may be too small; exit 1;};`
```

Comment:
I wrote a go language style function that can replace this shell
like this:
```go 
func DirUsage(dir string) (int64, error) {
	var usage int64
	fn := func(path string, d fs.DirEntry, err error) error {
		if err != nil {
			return err
		}
		info, err := d.Info()
		if err != nil {
			return err
		}

		st, ok := info.Sys().(*syscall.Stat_t)
		if !ok {
			usage += info.Size()
		} else {
			usage += st.Blocks * 512
		}
		return nil
	}
	if err := filepath.WalkDir(dir, fn); err != nil {
		return 0, err
	}

	return usage, nil
}
```

Reply:
nice，maybe not necessary， shell work good ，and keep sidecar image small